src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
Consider using a loop to create and print balls as it will make the code more scalable and reduce repetition. If you need to generate more balls in the future, you can simply change the loop's limit.

Reply:
Consider using a loop to create and print balls as it will make the code more scalable and reduce repetition. If you need to generate more balls in the future, you can simply change the loop's limit.